tools/gen_sig_info.py

```diff
@@ -357,6 +365,11 @@ def main(args):
 
   print('generating signatures ...')
   sig_info = {}
+  extract_sig_info(sig_info, {'WASMFS': 1,
+                              'JS_LIBRARIES': [],
+                              'USE_SDL': 0,
+                              'MAX_WEBGL_VERSION': 0,
+                              'AUTO_JS_LIBRARIES': 0}, cxx=True)
```

Comment:
Can the last 4 lines here be removed - are those not the default values? I don't see them set in the `extract_sig_info` lines below.